src/opt.cpp

```diff
@@ -619,6 +620,16 @@ void ispc::Optimize(llvm::Module *module, int optLevel) {
             }
         }
 #endif
+        // On ARM NEON targets LoadStoreVectorizer works great to combine
+        // multiple loads/stores into a single vector load/store as in #2052.
+        // However because of LoadStoreVectorizer after vector loads,
+        // elements are often extracted and processed individually using scalar operations.
+        // To vectorize these scalar operations, SLP vectorizer is used.
+        // Enabling it on other targets may be beneficial but require extensive testing.
+        if (ISPCTargetIsNeon(g->target->getISPCTarget())) {
+            optPM.addFunctionPass(llvm::LoadStoreVectorizerPass(), 325);
+            optPM.addFunctionPass(llvm::SLPVectorizerPass(), 326);
```

Comment:
No need to specify 326 after 325.

And you are not using these pass numbers for testing, so I would suggest just using the default.

The feature to specify pass number was design to pin the pass number for testing.

Reply:
Done